File: asp.net/sdk-lpl-mvc5/sdk-lpl-mvc5/Models/CreationCompteModel.cs
using System;

namespace sdk_lpl_mvc5.Models
{
    /// <summary>
    /// Modèle pour la création de compte
    /// Représente le modèle de données reçu lors de l'envoi d'une requete LPL de création de compte partenaire
    /// </summary>
    public class CreationCompteModel
    {
        public string CodeUtilisateur { get; set; }
        public string Mail { get; set; }
        public string Pseudo { get; set; }
        public string Password { get; set; }
        public string TypeAbonnement { get; set; }
        public DateTime DateExpiration { get; set; }
        public DateTime DateSouscription { get; set; }
        public double Tarif { get; set; }
        public int Statut { get; set; }
    }
}

File: nodejs/models/CreationCompteModel.js
/**
* Modèle pour la création de compte
* Représente le modèle de données reçu lors de l'envoi d'une requete LPL de création de compte partenaire
* @author  Bastien Caubet <bastien@nextinpact.com>, Luc Raymond <luc@nextinpact.com>
*/

module.exports = CreationCompteModel;

function CreationCompteModel(obj) {
    this.CodeUtilisateur = obj.CodeUtilisateur;
    this.Pseudo = obj.Pseudo;
    this.Mail = obj.Mail;
    this.Password = obj.Password;
    this.TypeAbonnement = obj.TypeAbonnement;
    this.DateSouscription = obj.DateSouscription;
    this.DateExpiration = obj.DateExpiration;
    this.Tarif = obj.Tarif;
    this.Statut = obj.Statut;
}